sql/core/src/test/resources/test-data/postgresql/onek.data

```diff
@@ -0,0 +1,1000 @@
+147	0	1	3	7	7	7	47	147	147	147	14	15	RFAAAA	AAAAAA	AAAAxx
```

Comment:
In the PR description, could you describe the location of these data sources, too?

Reply:
done.